<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExchangeWidget</class>
 <widget class="QWidget" name="ExchangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="selectedSourceDir">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="selectSourceDirButton">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Открыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendSelectedFile">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="sourceFiles">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="destFiles">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
